frontend/src/features/projects/erp/tabs/ErpTabs.tsx

```diff
@@ -42,17 +42,23 @@ const ErpTabs: React.FunctionComponent<IErpTabsProps> = ({
     <React.Fragment>
       <Tabs activeKey={currentTab} id="approvalTabs" onSelect={(key: string) => setCurrentTab(key)}>
         <Tab eventKey={SPPApprovalTabs.projectInformation} title="Project Information">
-          <ProjectInformationTab isReadOnly={isReadOnly} />
+          {currentTab === SPPApprovalTabs.projectInformation && (
```

Comment:
bootstrap actually render all tab content even when the tab is not displayed which is bad for formik performance.